extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -276,6 +274,24 @@ public function processCuratedContent( $data ) {
 		return $data;
 	}
 
+	public function processTrendingArticles( $data ) {
```

Comment:
I prefer to pollute this API which is going to be replaced anyway (some day, I hope!) and not our frontend. This way we can just swap this hacky thing to a new service.

IMO we should assume that there is a source with the correct data.